.pre-commit-config.yaml

```diff
@@ -16,6 +16,20 @@ repos:
               | ^poetry/core/_vendor
           )
 
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.910
+    hooks:
+      - id: mypy
+        pass_filenames: false
+        # files: |
```

Comment:
Let's go ahead and elide this entirely -- there should be no reason to directly exclude files in pre-commit at this time.